config/api.yml

```diff
@@ -84,6 +84,23 @@
       :get:
       - :name: read
         :identifier: availability_zone_show
+  :blueprints:
+    :description: Blueprints
+    :verbs: *70174834085860
+    :options:
+    - :collection
+    :klass: Blueprint
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: blueprints_list
```

Comment:
@imtayadeway do these need to match the feature id from db/fixtures/miq_product_features.yml ?